moped-database/migrations/1698070704301_backfill_confirmed_phase_dates/down.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE moped_proj_phases SET is_phase_start_confirmed = false WHERE phase_start <= current_date;
+UPDATE moped_proj_phases SET is_phase_end_confirmed = false WHERE phase_end <= current_date;
```

Comment:
actually, I don't think these down migrations will work, because the trigger exists that will automatically change these upon update. 

should i change this file to 

```
-- can't undo
select 0;
```

Reply:
I think one thing you could do here to work around that trigger is something like prepending this

```
ALTER TABLE moped_proj_phases DISABLE TRIGGER <trigger name>;
```

and then doing the updates, and then enabling it again

```
ALTER TABLE moped_proj_phases ENABLE TRIGGER <trigger name>;
```

Reply:
i'm cool with having an empty down migration on this one. If we're going to undo anything it will involve nuking the triggers entirely.

Thanks for that example, though, Frank. Trigger trickery is going to be increasingly necessary in the Moped DB!